app/views/admin/_team.erb

```diff
@@ -1,6 +1,6 @@
 <%= link_to t('team.add_team'), new_team_path, class: "govuk-button" %>
 
-<% teams.reverse.each do |team| %>
+<% teams&.each do |team| %>
```

Comment:
Did you mean to drop the `reverse`?

Reply:
Yeah its not needed and not sure what purpose it had before. The order of displaying teams doesn't matter

Reply:
Cool